frontend/app/app.html

```diff
@@ -5,28 +7,55 @@
         <meta charset="utf-8">
         <meta name="description" content="">
         <meta name="viewport" content="width=device-width, initial-scale=1">
-        <!--support file upload in non-html5 browsers-->
-        <script src="//cdnjs.cloudflare.com/ajax/libs/danialfarid-angular-file-upload/1.4.0/angular-file-upload-shim.min.js"></script>
-        <!--angular and its dependent packages-->
-        <script src="//ajax.googleapis.com/ajax/libs/angularjs/1.2.12/angular.js"></script>
-        <script src="//cdnjs.cloudflare.com/ajax/libs/danialfarid-angular-file-upload/1.4.0/angular-file-upload.min.js"></script>
-        <script src="//cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.10/angular-ui-router.min.js"></script>
-        <script src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.2.18/angular-resource.min.js"></script>
-        <script src="//d19d7ovrypufff.cloudfront.net/js/xeditable.min.js"></script>
-        <link rel="stylesheet" type="text/css" href="http://d19d7ovrypufff.cloudfront.net/css/xeditable.css">
+        <link rel="shortcut icon" href="http://static.wixstatic.com/ficons/c7673c_8e985095d4237d5e976870a290c8d7b7_fi.ico" type="image/x-icon">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
+        <link rel="stylesheet" href="https://s3-us-west-2.amazonaws.com/itreeware/planetLabStyle.css" type="type/css">
+ 
+        <title>PlanetLAB by Freedom Games</title>
+    </head>
+    <body>
+        <section>
+        <a ui-sref="userProfile">
+        <img  ng-src="https://s3-us-west-2.amazonaws.com/itreeware/PL--Header.png" alt="PlanetLAB |Freedom Games"   > </a>
+
+
+        <p style="float:right"> 
+                <a href="#" class="btn btn-default" ng-controller="LogOutCtrl" role="button" ng-click="logOut()">Log Out</a></p>
+        <hr>
+        <br>
+        <div ui-view> </div>  <!--All templateURL definitions have their content added to the DOM here.  
+                                The surrounding wrapper code persists throughout the planetApp -->
+        <hr>
+       
+        <br>
+        <div class="section s6">
+            <div >
+                <img class="tinyImg" src="https://s3-us-west-2.amazonaws.com/itreeware/FreedomGames_Logo.png" alt="Freedom Games Logo">
+                <div class="content">
+                    <p><b><a href="http://www.freedomgames.org" target="_blank">Freedom Games</a></b> is a design collaborative dedicated to excellence in teaching and learning. We partner with young artisans and teacher-leaders to create toys, games, and family media products that help communities learn together.</p>
+                    <h6>Planet Lab's first Minimum Viable Product (MVP) software will be published under the GNU General Public License (GPL).</h6>
```

Comment:
Stating the open source we are developing under was added when conversation came up from a possible grant or something a while back.   This is the footer from PlanetaryLaboratory.org.   I updated our app footer to reference the BSD 3-clause license.   TheresaStrepekOracle Middleware DeveloperMobile | Fusion | AngularJS 


-------- Original Message --------
Subject: Re: [Planet-Lab] Login Styling (#39)
From: Walt Askew <notifications@github.com>
Date: Sat, April 18, 2015 7:56 am
To: freedomgames/Planet-Lab <Planet-Lab@noreply.github.com>
Cc: "Tree Strepek (active)" <tree@iTreeware.com>

In frontend/app/app.html: > +
> +        <p style="float:right"> 
> +                <a href="#" class="btn btn-default" ng-controller="LogOutCtrl" role="button" ng-click="logOut()">Log Out</a></p>
> +        <hr>
> +        <br>
> +        <div ui-view> </div>  <!--All templateURL definitions have their content added to the DOM here.  
> +                                The surrounding wrapper code persists throughout the planetApp -->
> +        <hr>
> +       
> +        <br>
> +        <div class="section s6">
> +            <div >
> +                <img class="tinyImg" src="https://s3-us-west-2.amazonaws.com/itreeware/FreedomGames_Logo.png" alt="Freedom Games Logo">
> +                <div class="content">
> +                    <p><b><a href="http://www.freedomgames.org" target="_blank">Freedom Games</a></b> is a design collaborative dedicated to excellence in teaching and learning. We partner with young artisans and teacher-leaders to create toys, games, and family media products that help communities learn together.</p>
> +                    <h6>Planet Lab's first Minimum Viable Product (MVP) software will be published under the GNU General Public License (GPL).</h6>
 The current license is the BSD 3-clause. Do we need this line at all? —Reply to this email directly or view it on GitHub.